experiments/convert_dataset_parquet2jsonl.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+import logging
+
+import ray
+import draccus
+
+from operations.transform.format.parquet2jsonl import JsonlConversionConfig, convert_parquet_to_jsonl
+from marin.utils import fsspec_glob
+from marin.execution.executor import ExecutorStep, executor_main
+
+logger = logging.getLogger(__name__)
+
+@draccus.wrap()
+def convert_dataset_parquet2jsonl(cfg: JsonlConversionConfig) -> bool:
+    """
+    Convert any Parquet-based dataset under cfg.input_path (including nested directories)
+    into JSONL (zst-compressed) under cfg.output_path, preserving subdirectory structure.
+    """
+    logger.info(f"Starting conversion from {cfg.input_path} to {cfg.output_path}")
+    # Recursively locate all Parquet files
+    parquet_files = fsspec_glob(os.path.join(cfg.input_path, "**", "*.parquet"))
+    if not parquet_files:
+        logger.warning(f"No parquet files found under {cfg.input_path}")
+        return True
+
+    futures = []
+    for path in parquet_files:
+        # Compute the relative path under the input root
+        rel_path = os.path.relpath(path, start=cfg.input_path)
+        # Replace extension and build output path
+        out_rel = rel_path.replace(".parquet", ".jsonl.zst")
+        out_path = os.path.join(cfg.output_path, out_rel)
+        futures.append(convert_parquet_to_jsonl.remote(path, out_path))
+
+    if futures:
+        ray.get(futures)
+    logger.info(f"Completed conversion of {len(futures)} shards")
+    return True
+
+if __name__ == "__main__":
+    if len(sys.argv) != 2:
+        print("Usage: python convert_dataset_parquet2jsonl.py <dataset_relative_path_under_raw>")
+        sys.exit(1)
```

Comment:
I don't think we should make this a command-line utility with arguments, but rather have a function that people can call...and we write down in code all the invocations of that function (which you have to run anyway).  That's the Marin way.